content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+_db_id: 499
```

Comment:
Database ids get autogenerated.  It's better not to fill them in

Reply:
Cool, thought we where just replacing the content of one with the other. I'll revert and add new stuff separately.